docs/content/usage/executor.en.md

```diff
@@ -59,10 +59,12 @@ Status code distribution:
   [200]	10000 responses
 
 ```
-While the load is being generated, we will watch the HorizontalPodAutoscaler and how it scales over period of time. As you can notice, the number of pods is scaled from 1 to 3 after the load rises from 8 - 103%. After the load generator stops, it takes a few iterations to scale down from 3 to 1 pod. 
+While the load is being generated, we will watch the HorizontalPodAutoscaler and how it scales over period of time. As you can notice, the number of pods is scaled from 1 to 3 after the load rises from 8 - 103%. After the load generator stops, it takes a few iterations to scale down from 3 to 1 pod.
+
+You will notice that the scaling up and down has different behaviour in terms of response time. This behaviour is governed by the frequency at which the controller watches (which defaults to 30s) and parameters set on controller-manager for upscale/downscale delay. More details can be found [here](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-cooldowndelay)
```

Comment:
Can you include the actual one minute number for upscale/downscale delay here? Or is it more complicated than that?

Reply:
Default upscale delay is 3 minute - which means between two scale up events there will be 3 minutes. Similarly, default downscale delay is 5 minutes.

I think the overall behaviour is complex, so leading to current documentation page seemed a better idea.

Reply:
Most people go through this page, run a load generator for 30 seconds, and wonder why their autoscaler did not kick in.  You should include at least something here about the initial autoscaler delay.  It is also the reason people will need to set minscale to handle any load they expect in the first 1 minute (or 3 minutes, or... whatever the delay is for the first scale up).